pkg/node/volumemgr.go

```diff
@@ -1110,6 +1110,12 @@ func (m *VolumeManager) createEventsForDriveUpdates(updates *driveUpdates) {
 			m.createEventForDriveStatusChange(
 				updDrive.CurrentState, updDrive.PreviousState.Spec.Status, updDrive.CurrentState.Spec.Status)
 		}
+		if updDrive.CurrentState.Spec.Status == apiV1.DriveStatusOffline {
```

Comment:
Is it possible that we get two same events the one generated in line 1111 and this new one?

Reply:
For normal flow is should not be possible, however to to be 100% safe I've changed the code to handle this situation. Thank you!